src/frontend/planner_test/src/lib.rs

```diff
@@ -416,6 +416,8 @@ impl TestCase {
                     source_schema,
                     source_watermarks,
                     append_only,
+                    cdc_source_name,
+                    external_table_name,
```

Comment:
Will these fields always set to `Some` or `None` simultaneously? If so, is it possible to combine them?